src/components/StakingVaults/EarnOpportunities.tsx

```diff
@@ -36,7 +36,7 @@ export const EarnOpportunities = ({ assetId }: EarnOpportunitiesProps) => {
   } = useWallet()
   const asset = useAppSelector(state => selectAssetById(state, assetId))
   const { vaults } = useVaultBalances()
-  const { data: foxyBalancesData } = useFoxyBalances()
+  const { data: foxyBalancesData } = useFoxyBalances({ accountNumber: 0 })
```

Comment:
When we add enumeration, `useFoxyBalances` will eventually be able to return all balances, so we will be able to ditch this